airgun/views/repository.py

```diff
@@ -274,7 +270,6 @@ class FileRepository(View):
         upstream_url = EditableEntry(name='Upstream URL')
         verify_ssl = EditableEntryCheckbox(name='Verify SSL')
         upstream_authorization = AuthorizationEntry(name='Upstream Authorization')
-        publish_via_http = EditableEntryCheckbox(name='Publish via HTTP')
         unprotected = EditableEntryCheckbox(name='Unprotected')
```

Comment:
Are file and yum repos the only repository types that support unprotected?

Reply:
Just checked 6.12 and they are the only repo types with unprotected as an option.